scriptmodules/supplementary/webgui.sh

```diff
@@ -0,0 +1,38 @@
+
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="webgui"
+rp_module_desc="RetroPie WebGUI on port 8080"
+rp_module_section="exp"
+
+function depends_webgui() {
+    getDepends sqlite3 php5 php5-sqlite
+}
+
+function install_bin_webgui() {
+    gitPullOrClone "$md_inst" "https://github.com/daeks/RetroPie-WebGui"
+}
+
+function configure_webgui() {
+    killall php
+    php -S "$(hostname -I | xargs):8080" -t "$md_inst" > /dev/null 2>&1 &
```

Comment:
I would not kill/ start the service here like that. I would create a little start script to avoid having to do this and keep it simple in /etc/rc.local - you can also then save the PID for starting/stopping etc

Also `ip route get 8.8.8.8 | head -1 | cut -d' ' -f8` is more reliable for getting the interface ip.